crates/sui-core/src/unit_tests/authority_tests.rs

```diff
@@ -419,10 +419,10 @@ async fn test_transfer_package() {
         package_object_ref,
         gas_object.compute_object_reference(),
     );
-    let result = authority_state
+    authority_state
         .handle_transaction(transfer_transaction.clone())
-        .await;
-    assert_eq!(result.unwrap_err(), SuiError::TransferUnownedError);
+        .await
+        .unwrap_err();
```

Comment:
Can we not test the error kind at this point?

Reply:
unfortunately the error type that comes out of here is a SuiError and that context is lost now.